cedar-policy-validator/src/schema/namespace_def.rs

```diff
@@ -735,4 +666,12 @@ impl ValidatorNamespaceDef {
     pub fn namespace(&self) -> &Option<Name> {
         &self.namespace
     }
+
+    fn prefix_namespace(ns: Option<Name>, name: Name) -> Name {
```

Comment:
I'd like a more explicit name here. It important this this _doesn't_ apply the prefix if the name is already qualfied, so `prefix_namespace_if_unqualified` or something. Kind of verbose, but it makes it easier to understand that it's doing the correct thing when you call it.

Reply:
Done.